packages/react-server/src/ReactFizzServer.js

```diff
@@ -194,6 +196,8 @@ export opaque type Request = {
   partialBoundaries: Array<SuspenseBoundary>, // Partially completed boundaries that can flush its segments early.
   // onError is called when an error happens anywhere in the tree. It might recover.
   onError: (error: mixed) => void,
+  // getErrorHash is used in production primarily to avoid leaking internals, secondarily to save bytes. In production we'll send the error hash.
+  getErrorHash: (error: mixed) => string,
```

Comment:
Can we instead use the return value of `onError` and if it's undefined we blank it out in PROD or pass it through? That way we don't need to introduce another API and deal with conflicts, and you have the option to hash or encrypt it in DEV too.